govstack_api/building_blocks/bb_digital_registries/registries/base_registry.py

```diff
@@ -1,17 +1,95 @@
+import base64
+import json
+import xml.etree.ElementTree as ET
 from typing import Protocol
+from xml.dom import minidom
+
+
+class RegistryProtocol(Protocol):
```

Comment:
You can just keep "Registry" as a name. or change it to RegistryType

Reply:
Okay